quadkey.c

```diff
@@ -24,7 +24,9 @@ typedef unsigned int uint32;
 #define INV_WM_RANGE (1.0/WM_RANGE)
 #define WM_MAX (M_PI*WEBMERCATOR_R)
 
-static inline uint64 xy2quadint(uint64 x, uint64 y) {
+static inline uint64
+xy2quadint(uint64 x, uint64 y)
+{
```

Comment:
It's great that you're making style consistent, but this could lead us to some [internal war](https://en.wikipedia.org/wiki/Indent_style) 😝 

OK, well, since you took the initiative I'll let you go without a fight.